OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -29,15 +29,7 @@ namespace OpenDreamRuntime.Procs.Native {
     /// Note that this proc container also includes global procs which are used to create some DM objects,
     /// like filter(), matrix(), etc.
     /// </remarks>
-    internal static class DreamProcNativeRoot {
-        [DreamProc("abs")]
-        [DreamProcParameter("A", Type = DreamValueTypeFlag.Float)]
-        public static DreamValue NativeProc_abs(NativeProc.Bundle bundle, DreamObject? src, DreamObject? usr) {
-            bundle.GetArgument(0, "A").TryGetValueAsFloat(out var number);
-
-            return new DreamValue(Math.Abs(number));
-        }
-
+    static class DreamProcNativeRoot {
```

Comment:
Don't remove `internal`
```suggestion
    internal static class DreamProcNativeRoot {
```